07-modeling.Rmd

```diff
@@ -203,14 +207,14 @@ recs_in %>%
   theme_bw() 
 ```
 
-The model is fit below with electricity expenditure as the outcome. Some textbooks use the term simple linear regression when a linear model has one continuous independent variable.
+The model is fit below with electricity expenditure as the outcome. Some textbooks use the term simple linear regression when a linear model has one continuous independent variable. 
```

Comment:
Can we expand on "simple linear regression"? Or define it up top? I am not sure what it means to have it here.

Reply:
I agree that it is an odd placement to introduce this.  I ended up just removing this sentence, but we can add it back in if others think it is necessary to have.